tests/caasp/stack_bootstrap.pm

```diff
@@ -130,8 +130,17 @@ sub kubectl_config {
 }
 
 sub run {
-    assert_screen 'velum-bootstrap-page', 90;
     barrier_wait {name => "WORKERS_INSTALLED", check_dead_job => 1};
```

Comment:
Please don't switch order here, otherwise we won't know how long it takes to load bootstrap page.

Reply:
ok, I did that on purpose actually in order to benefit from the time during the `barrier_wait`, but I see your point ;) I will re-order it.